webapp/channels/src/components/admin_console/boolean_setting.tsx

```diff
@@ -15,83 +15,86 @@ type Props = {
     onChange: (id: string, foo: boolean) => void;
     trueText?: React.ReactNode;
     falseText?: React.ReactNode;
-    disabled: boolean;
+    disabled: boolean | undefined;
```

Comment:
are we certain it will be undefined? because I see its default value is false

Reply:
It's undefined in a ton of upstream components that utilze this, which is why I made it this way. I'll adjust it to be `disabled?: boolean;` so it's more clear.